packages/nextjs-webpack-plugin/src/nextjs-webpack-bundle-analysis/nextJSWebpackBundleAnalysisPlugin.ts

```diff
@@ -43,7 +43,7 @@ export const nextJSWebpackBundleAnalysisPlugin: ExtendedBAUploadPlugin<{
           stage: webpack.Compilation.PROCESS_ASSETS_STAGE_REPORT,
         },
         async () => {
-          output.setBundleName(output.bundleName);
+          output.setBundleName(output.originalBundleName);
```

Comment:
is this a function to invoke like output.originalBundleName()?

Reply:
so with a `get` method you don't need to actually call it, it handles that under the hood for you.